sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -1460,6 +1460,21 @@ Accept wildcard characters: False
 
 ```
 
+
+### -ClearGroupId
```

Comment:
>ClearGroupId [](http://example.com/codeflow?start=5&length=12)

you need to add this to the SYNTAX section as well. Scroll to top of page.

Reply:
added